src/TodoInputForm.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useCallback } from "react";
+import styled from "styled-components";
+
+const TitleContents = styled.section`
+  padding-bottom: 15px;
+  border-bottom: 1px solid lightgrey;
+`;
+const TodoTitle = styled.h2`
+  padding-left: 20px;
+`;
+const TodoInputItems = styled.form`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+const TodoInput = styled.input`
+  border: 1px solid lightgrey;
+  border-radius: 10px;
+  padding: 15px;
+  width: 230px;
+  :focus {
+    outline: none;
+    transition: 0.2s;
+    box-shadow: 0px 0px 5px lightgrey;
+  }
+`;
+const TodoInputBtn = styled.button`
+  background: none;
+  border: none;
+  font-size: 30px;
+  margin-left: 10px;
+  cursor: pointer;
+  border-radius: 30px;
+  :hover {
+    color: grey;
+    transition: 0.2s;
+  }
+`;
+
+const TodoInputForm = ({ todoList, setTodoList }) => {
+  const [newText, setNewText] = useState("");
+
+  const _changeText = useCallback(({ target }) => {
```

Comment:
이벤트 핸들러의 이름은 보통 `handle + 대상 + 동작` 형식으로 짓는다고 합니다~

```suggestion
  const handleTextChange = useCallback(({ target }) => {
```

Reply:
그렇군요! 네이밍에 더 신경쓰겠습니다!!